### ISSUE TITLE
Submodules not being released with tags

### ISSUE THREAD

**Issue by pete-woods** [alexedwards/scs#203]
Dependabot / renovate have trouble with the sub-modules in this module, seemingly because they are not released / tagged.

When you have submodules, usually it's easiest to follow some rules:
- Have all submodules follow the same major version.
- Tag all submodules for release at the same time as the main module:
  - e.g. `v2.8.0` for the main module
  - and `badgerstore/v2.8.0`, `boltstore/v2.8.0`, `bunstore/v2.8.0`, etc for the sub modules.

See an example of how the tagging is supposed to be done here: https://github.com/hbjydev/go-submods-example/tags

At the moment, the store submodules all appear to be v1 modues, and also not tagged, so it really confuses any tools that want to automatically update them, as those tools want to pull down the latest v1 SCS, which is `v1.4.1`, and things go wrong.

I don't know what strategy you prefer to employ here, but would really appreciate something being figured out. Also as I'm using this library I'm willing to help out in any way I can.

**Comment by deltamualpha** [comment]
Yeah, I had to just pull the `redisstore` module directly into my codebase because when go's module resolution tried to work out how to use `github.com/alexedwards/scs/redisstore` it always ended up trying to load `github.com/alexedwards/scs v1.4.1`, the packages of which are not compatible with `github.com/alexedwards/scs/v2`.

**Comment by dropwhile** [comment]
This is what I have in my `go.mod` right now, and it seems to be working:
```
	github.com/alexedwards/scs/goredisstore v0.0.0-20251001152029-20477ae1fec6
	github.com/alexedwards/scs/v2 v2.9.0
```

and here is what an import example looks like
```
import (
    // other stuff...

    "github.com/alexedwards/scs/goredisstore"
    "github.com/alexedwards/scs/v2"
    "github.com/redis/go-redis/v9"
)
```

**Comment by jum** [comment]
This article is a bit old, but describes the process:

[go-modules-by-example/index/blob/master/009_submodules](https://github.com/go-modules-by-example/index/blob/master/009_submodules/README.md)

If I understand that properly, we would need to have individual tags in the form:

```
git tag goredisstore/v2.0.0
```

Does that sound sensible?

**Comment by pete-woods** [comment]
That's how it's supposed to work, yes. I did try to describe it fully in the issue.

The repo I linked to also does the correct thing -- you can look at its tags.

In deopwhile's usage there `github.com/alexedwards/scs/goredisstore v0.0.0-20251001152029-20477ae1fec6`, you can see you're plainly having to use a v0 SHA for the submodule. Instead of a proper release, like the main scs module.

Basically i think the repo needs a GitHub Action that runs:

```
git tag v2.x.x
git tag boltstore/v1.x.x
git tag redisstore/v1.x.x
```

That sort of thing. So the maintainer can run it whenever a new release is made.

There's a good chance there's an off the shelf go tagging action
